.github/workflows/static_analysis.yml

```diff
@@ -27,3 +28,59 @@ jobs:
           distribution: 'zulu'
       - name: Lint
         run: ./gradlew lint
+
+  diffuse:
+    name: Diffuse AAR Analysis
+    runs-on: macOS-11
+    steps:
+      # Set up environment to assemble SDK
+      - name: Set up Java 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'zulu'
+
+      # Assemble artifacts from main branch
+      - name: Checkout Main Branch
+        uses: actions/checkout@v2
+        with:
+          ref: main
+      - name: Assemble Release AAR on Main Branch
+        run: ./gradlew clean assembleRelease
+      - name: Upload Main Branch Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: main-aar
+          path: |
+            Core/build/outputs/aar/Core-release.aar
+            Card/build/outputs/aar/Card-release.aar
+
+      # Assemble artifacts from current branch
+      - name: Checkout Current Branch
+        uses: actions/checkout@v2
+      - name: Assemble Release AAR on Current Branch
+        run: ./gradlew clean assembleRelease
+      - name: Upload Current Branch Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: current-aar
+          path: |
+            Core/build/outputs/aar/Core-release.aar
+            Card/build/outputs/aar/Card-release.aar
+
+      # Set up Diffuse
+      - name: Install Diffuse
+        run: brew install JakeWharton/repo/diffuse
+      - name: Download Workflow Artifacts 
+        uses: actions/download-artifact@v2
+      - name: Set up Java 11
```

Comment:
This is a diffuse bug - currently aar comparisons can only be done using Java 11